### ISSUE TITLE
E3-9: Write MINING.md onboarding guide

### ISSUE THREAD

**Issue by jkbrooks** [jkbrooks/autoresearch_network#17]
## E3-9: Write `MINING.md` onboarding guide

**Parent Epic:** #8 — Miner Implementation  
**Dependencies:** All other E3 tickets (this documents the finished miner)  
**Implementation order:** 9 of 9 (MVP) — write after all miner code is merged

---

### What This Is

A single document that takes a new miner from zero to earning emissions in under 30 minutes, with copy-pasteable commands for every step. This is the primary onboarding surface for new miners and a key credibility artifact for investors and partners.

### Where It Goes

`docs/MINING.md`

### Target Audience

A person who:
- Has a Linux machine with an NVIDIA GPU (≥8GB VRAM)
- Has used a terminal before
- Has never used Bittensor
- May or may not have an LLM API key
- Wants to start mining quickly without reading the full codebase

### Required Sections

**1. Requirements**
- OS: Ubuntu 22.04+ recommended (any Linux with CUDA works)
- GPU: minimum 8GB VRAM (SMALL tier), recommended 24GB+ (LARGE tier)
- Python: 3.10+
- `uv`: latest (install command provided)
- Disk: ~10GB free for data cache + model checkpoints
- Network: stable internet for chain sync and challenge/response

**2. Install**
```bash
git clone https://github.com/jkbrooks/autoresearch_network.git
cd autoresearch_network
pip install -e .
uv sync
```
Explain what each command does in one sentence.

**3. One-Time Data Setup**
```bash
uv run prepare.py
```
Explain: downloads training data from Hugging Face, trains BPE tokenizer, caches to `~/.cache/autoresearch/`. Takes ~2 minutes. Only needed once.

**4. Create Wallet**
```bash
btcli wallet new_coldkey --wallet.name my-miner
btcli wallet new_hotkey --wallet.name my-miner --wallet.hotkey default
```
Explain coldkey vs hotkey. Emphasize: **backup your seed phrases now.** Show where files are stored (`~/.bittensor/wallets/`).

**5. Get Test TAO**
- Link to Bittensor Discord: https://discord.gg/bittensor
- Channel: `#faucet` or `#testnet-faucet` (explain you need to ask)
- How much: ~100 test TAO is sufficient

**6. Register on Testnet**
```bash
btcli subnet register --netuid <NETUID> --network test \
  --wallet.name my-miner --wallet.hotkey default
```
Replace `<NETUID>` with the actual testnet subnet ID (document it here once deployed).

**7. Configure LLM Mutations (Optional)**
```bash
# For Anthropic Claude (recommended)
export ANTHROPIC_API_KEY="sk-ant-..."

# For OpenAI
export OPENAI_API_KEY="sk-..."
```
Explain: with an API key, the miner uses an LLM agent to propose creative training improvements. Without one, the miner uses a built-in structured mutation engine that cycles through known high-signal hyperparameter changes. Both work. The LLM path produces better results but costs money.

**API Cost Budgeting:**
| Provider | Cost per experiment | Cost per hour (12 exp/hr) | Cost per day (24hr) |
|----------|-------------------|---------------------------|---------------------|
| Anthropic Claude Opus | ~$0.06 | ~$0.72 | ~$17 |
| OpenAI GPT-4o | ~$0.02 | ~$0.24 | ~$6 |
| Local/compatible endpoint | $0 | $0 | $0 |

Explain: each experiment sends ~21K tokens (prompt + response). Miners should budget for this. Using a local LLM via `--mutation-provider openai-compatible --mutation-base-url http://localhost:8000` eliminates API costs entirely.

**8. Run the Miner**
```bash
python neurons/miner.py \
  --netuid <NETUID> \
  --network test \
  --wallet.name my-miner \
  --wallet.hotkey default \
  --mutation-provider anthropic \
  --logging.debug
```
Explain each flag. Recommend running under `tmux` or `screen` for persistence.

Without API key:
```bash
python neurons/miner.py \
  --netuid <NETUID> \
  --network test \
  --wallet.name my-miner \
  --wallet.hotkey default \
  --logging.debug
```

**9. Verify You're Earning**
```bash
btcli wallet overview --wallet.name my-miner --network test
```
What to look for: `EMISSION` column should be non-zero after 1–2 tempos (~2–4 hours). `INCENTIVE` shows your score relative to other miners. `RANK` shows your position.

**10. Troubleshooting**
| Problem | Solution |
|---------|----------|
| Health check: no GPU detected | Install NVIDIA drivers + CUDA toolkit. Verify with `nvidia-smi`. |
| Health check: `uv` not found | `curl -LsSf https://astral.sh/uv/install.sh \| sh` |
| Health check: data cache missing | Run `uv run prepare.py` |
| OOM crash during training | Your GPU has insufficient VRAM for the current model config. The structured mutation strategy will try smaller configurations automatically. |
| `val_bpb` not improving | Normal at first. The LLM strategy explores more creatively. Ensure your API key is set. |
| Emissions are zero after 4+ hours | Check that your validator is running and scoring. Check `btcli subnet show --netuid <NETUID>`. |

### Acceptance Criteria

- [ ] All commands in the doc are verified to work on testnet (run through the entire guide manually)
- [ ] A team member unfamiliar with Bittensor can follow it without asking questions
- [ ] API cost budgeting section is accurate and includes the local-LLM option
- [ ] Troubleshooting section covers the top 5 failure scenarios seen during testing
- [ ] Document renders correctly as GitHub markdown

### Test Expectations

No code tests — this is a documentation ticket. Acceptance is verified by a manual walkthrough.
